src/pages/Auth/RegisterYouth.tsx

```diff
@@ -0,0 +1,149 @@
+import { yupResolver } from "@hookform/resolvers/yup";
+import React, { useEffect, useState } from "react";
+import { useForm } from "react-hook-form";
+import { Link, useNavigate } from "react-router-dom";
+import * as Yup from "yup";
+import { useAuth } from "../../AuthContext";
+import FormError from "./FormError";
+
+interface FormValues {
+  firstName: string;
+  lastName: string;
+  birthDate: Date;
+  ssn: string;
+  email: string;
+  password: string;
+  program: string;
+  active: boolean;
+  confirmPassword: string;
+  attached_forms?: string[];
+  attended_events?: string[];
+}
+
+const schema = Yup.object().shape({
+  firstName: Yup.string().required("First Name is required"),
+  lastName: Yup.string().required("First Name is required"),
+  email: Yup.string()
+    .email("Invalid email address")
+    .required("Email is required"),
+  password: Yup.string()
+    .min(5, "Password must be at least 5 characters")
```

Comment:
Same thing to what sam said, make sure requirements match up. Additionally, seems like there's some overlap between requirements here and above. Try to stick to DRY: Don't Repeat Yourself. See if some of this can be simplified :)